superset-frontend/.eslintrc.js

```diff
@@ -226,6 +227,7 @@ module.exports = {
     'no-mixed-operators': 0,
     'no-multi-assign': 0,
     'no-multi-spaces': 0,
+    'no-nested-ternary': 0,
```

Comment:
Disabling as we have seen fairly often nested-ternary in TypeScript `extends`. It should not be that hard to understand the code with `prettier`'s help.

Reply:
nested ternary was used in this code:

```ts
        const controlName =
          typeof item === 'string'
            ? item
            : item && 'name' in item
            ? item.name
            : null;
```

Reply:
`'name' in item` is needed for type inference so to avoid forced type conversion as `item` can be of many types, otherwise this could also be as simple as `item?.name || item || null`.

Here's the error with your approach:

<img width="1143" alt="Screen Shot 2021-02-25 at 6 10 55 PM" src="https://user-images.githubusercontent.com/335541/109253871-95b66580-77a5-11eb-93fc-be914d8fe470.png">